components/automate-cli/pkg/verifyserver/constants/skipmessages.go

```diff
@@ -0,0 +1,13 @@
+package constants
+
+const (
+	SKIP_CERT_TEST_MESSAGE                   = "- Using Default certificates\n- No Custom certificates provided"
+	SKIP_NODE_CERT_TEST_MESSAGE              = "- Using Default certificates\n- No Custom certificates provided"
```

Comment:
```suggestion
	SKIP_CERT_TEST_MESSAGE                   = "- Using Default certificates\n- No custom certificates provided"
	SKIP_NODE_CERT_TEST_MESSAGE              = "- Using Default certificates\n- No custom certificates provided"
```

Reply:
I have capitalised it since its an Important word